src/rules/__tests__/valid-title.test.ts

```diff
@@ -31,7 +118,7 @@ ruleTester.run('title-must-be-string', rule, {
     },
     {
       code: 'xdescribe(skipFunction, () => {});',
-      options: [{ ignoreTypeOfDescribeName: true }],
+      options: [{ ignoreTypeOfDescribeName: true, disallowedWords: [] }],
```

Comment:
we do we have to define it here if it's the default? type error?

Reply:
Yeah it seems that if we don't have it in at least one of the test cases that have `options`, TS gets snippy.

There's a couple of different ways around it, but they've all got pros & cons: primarily we could define an actual interface for the options, but that'd also require us to double type the message ids since we don't have [partial type arguments](https://github.com/microsoft/TypeScript/issues/26242), and then that'd require us to provide a default value in the actual function (as in, in the parameter in the `create` function).

This made TS happy, so I just decided to go with it :)